Automattic/wp-calypso
# Make the login form "Return" link conditional on there being somewhere to return.

This pull request addresses/fixes #13785 by removing the "Return" link when there's nothing in `window.history`:
 
Login page without browser history:

<img width="504" alt="Login page without browser history" src="https://cloud.githubusercontent.com/assets/352291/25837355/cbd13c38-34d0-11e7-9e8a-344e4e6ef943.png">

Login page with browser history:

<img width="506" alt="Login page with browser history" src="https://cloud.githubusercontent.com/assets/352291/25837357/ce734b0c-34d0-11e7-9f31-c817590ca093.png">

 
#### Testing instructions
 
1. Run `git checkout fix/13785-login-return-link` and start your server, or open a [live branch](https://calypso.live/?branch=fix/13785-login-return-link)
2. Open the [`Login` page](http://calypso.localhost:3000/login)
3. Check that the "Return" link shows if your browser back button is active, and doesn't show if they button isn't active.
 
#### Reviews
